tools/docker-network/prometheus.yml

```diff
@@ -6,3 +6,5 @@ scrape_configs:
           - peer_master:9311
           - faucet:9311
           - peer_master2:9311
+          - peer_nomana:9311
+          - peer_replica:9311
```

Comment:
Instead of naming them statically maybe you can try something like this https://stackoverflow.com/questions/70803245/prometheus-service-discovery-with-docker-compose